pkg/authorizations/authorizations.go

```diff
@@ -0,0 +1,81 @@
+package authorizations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+	fungibletypes "github.com/zeta-chain/zetacore/x/fungible/types"
+	lightclienttypes "github.com/zeta-chain/zetacore/x/lightclient/types"
+	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func GetRequiredPolicy(msgURL string) authoritytypes.PolicyType {
+	if CheckPolicyList(msgURL, OperationalPolicyMessageList) {
+		return authoritytypes.PolicyType_groupOperational
+	}
+	if CheckPolicyList(msgURL, EmergencyPolicyMessageList) {
+		return authoritytypes.PolicyType_groupEmergency
+	}
+	if CheckPolicyList(msgURL, AdminPolicyMessageList) {
+		return authoritytypes.PolicyType_groupAdmin
+	}
+	return authoritytypes.PolicyType_emptyPolicyType
+}
+
+func CheckPolicyList(msgURl string, msgList []string) bool {
+	for _, policy := range msgList {
+		if policy == msgURl {
+			return true
+		}
+	}
+	return false
+}
+
+var OperationalPolicyMessageList = []string{
```

Comment:
I'm thinking about defining a type `PolicyTable` containing a list [url,policy] tuples with crud methods, so it can be easily update, even if we add a new policy type and can be used once we persist the table in the state